service/history/mutableStateBuilder.go

```diff
@@ -2992,6 +3004,7 @@ func (e *mutableStateBuilder) AddContinueAsNewEvent(
 		e.logger,
 		e.namespaceEntry,
 		timestamp.TimeValue(continueAsNewEvent.GetEventTime()),
+		command.GetBackoffStartInterval(),
```

Comment:
firstWorkflowTaskBackoff *time.Duration should not be part of the initialization logic